src/main/java/seedu/address/model/person/Person.java

```diff
@@ -52,6 +52,9 @@ public Address getAddress() {
         return address;
     }
 
+    public int getId() {
```

Comment:
Why is this code here?

Reply:
removed

Reply:
this is for the AddDriverCustomer code.

Reply:
I see, you don't need this code actually.